src/modules/universal-discovery/components/finder/finder.component.js

```diff
@@ -65,6 +65,45 @@ export default class FinderComponent extends Component {
         return { locationsMap: this.locationsMap, activeLocations: this.activeLocations };
     }
 
+    /**
+     * Loads starting location data (if there is a need) and loads root locations
+     *
+     * @method handleRootLocations
+     * @memberof FinderComponent
+     */
+    handleRootLocations() {
+        // Starting location is already loaded or its default
+        if (this.startingLocation.id === this.props.startingLocationId) {
```

Comment:
Please define a const with the root id, and check if `this.props.startingLocationId === ROOT_LOCATION_ID`
`const ROOT_LOCATION_ID = 1`